DOCS-7334 API Groups (#106)
mulesoft/docs-exchange: modules/ROOT/pages/about-my-applications.adoc

:imagesdir: ../assets/images/

> The My Applications area in the Exchange details view for an asset lists

Reviewer: ```suggestion
The *My Applications* area in the Exchange details view for an asset lists
```

> applications that hold a contract between a user and an API or an API Group.

Reviewer 2: that hold a contract ~between a user and~ *with* an API or an API Group.

_(I think "between a user" sounds weird, the contract actually is between the application itself and the API/APIGroup instance)_

Author: True, that's better. Fixed.

> Applications in My Applications are registered using API Manager in Anypoint Platform.

Reviewer: ```suggestion
Applications in *My Applications* are registered using API Manager in Anypoint Platform.
```

To register a client application to an existing API or API Group, the client application
must first request access. When the request is approved by the API or API Group owner, a
contract is created between the client application and the API or API Group, and the client
application is registered.

If you don't have an application to link to the API or API Group, you can create the application

> in the Request Access menu. You can then click Request Access to connect your

Reviewer: ```suggestion
in the *Request Access* menu. You can then click *Request Access* to connect your
```

application and the API or API Group. Some APIs and API Groups require selecting a service-level agreement

> (SLA), accepting terms and conditions, or both.

Reviewer: terms and conditions or both?

Author: SLA, or T&C, or both SLA and T&C. Updated for clarity.

APIs and API Groups that are protected by a client ID enforcement policy require client
applications to provide a client ID and optional client secret. The client ID and client
secret credentials are automatically created when the client application is
registered.

An API owner can use API Manager to view all API contracts ever created for an

> API or API Group instance. An API consumer can use Exchange's *My Applications* page to view

Reviewer: ```suggestion
API or API Group instance. An API consumer can use the Exchange *My Applications* page to view
```

> the API contracts created between an API instance or API Group instance and that consumer's client

Reviewer 2: the ~API~ contracts created between

We've never used the term "API contract" and at this point I'm not sure what it would mean. We tend to use "standalone api contract" or "api group contract", but because we are talking in general I would just use "contracts"

Author: Agreed. Fixed.
